toast-tk-web-agent/src/main/java/io/toast/tk/agent/web/rest/AsyncHttpClientProvider.java

```diff
@@ -1,15 +1,28 @@
 package io.toast.tk.agent.web.rest;
 
-import com.google.common.base.Strings;
-import com.google.inject.Inject;
-import com.google.inject.Provider;
-import io.toast.tk.agent.config.AgentConfig;
-import org.asynchttpclient.AsyncHttpClientConfig;
+import java.security.cert.CertificateException;
+
+import javax.net.ssl.SSLException;
+
+import org.apache.http.conn.ssl.TrustSelfSignedStrategy;
+import org.apache.http.ssl.SSLContextBuilder;
 import org.asynchttpclient.DefaultAsyncHttpClient;
 import org.asynchttpclient.DefaultAsyncHttpClientConfig;
+import org.asynchttpclient.DefaultAsyncHttpClientConfig.Builder;
 import org.asynchttpclient.Realm;
+import org.asynchttpclient.Realm.AuthScheme;
+import org.asynchttpclient.netty.ssl.InsecureTrustManagerFactory;
 import org.asynchttpclient.proxy.ProxyServer;
 
+import com.google.common.base.Strings;
+import com.google.inject.Inject;
+import com.google.inject.Provider;
+
+import io.netty.handler.ssl.SslContext;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'io.netty.handler.ssl.SslContext'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)